### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by Luchy-B**
- Add linters
- Add JS file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,24 +4,37 @@
...
+                            <li onclick="hide()" class="style-li"><a href="#contactPage">Contact</a></li>
+                            <li id="messge"><img src="images/Envelop.png" alt="Envelop"></li>
+                        </ul>                          
+                  </div> 
+                    <ul class="dskNavList">
+                        <li class="liList"><a href="#portfolioPage">Portfolio</a></li>
+                        <li class="liList"><a href="#aboutPage">About</a></li>
+                        <li class="liList"><a href="#contactPage">Contact</a></li>
+                        <li class="liList"><img src="images/Envelop.png" alt="Envelop"></li>
+                    </ul>
```
Comment (ShadyShawkat) [inline index.html]: - Please make sure to follow the design guidelines as much as possible, which include:
   - [x] Aligning the navigation links to the left of the mobile menu.
   - [x] Separating each link with a horizontal line to match the given design.
   - [x] Aligning the "close" button to the right of the mobile menu.
   - [x] Using the given font family and styling to match the given design.

| Your design | Figma design |
| --- | --- |
| image | image |

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      "import/extensions": [ 1, {
+        "js": "always", "json": "always"
+      }]
+    },
+    "ignorePatterns": [
+      "dist/",
+      "build/"
+    ]
+  }
+
```
Comment (ShadyShawkat) [inline .eslintrc.json:26]: - [x] Please make sure to set up the EsLint linters correctly. Kindly update the `./github/workflows/linters.yml` file using [this](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) configuration from the [linters setup for HTML, CSS, and JavaScript](https://github.com/microverseinc/linters-config/tree/master/html-css-js) projects.

- This is a required change to use the correct linters setup. However, in the future, setting up the inters incorrectly can result in the invalidation of your submission.

File: index.html [hunk 3]
```diff
@@ -4,24 +4,37 @@
...
-                        <li>About</li>
-                        <li>Contact</li>
-                        <li><img src="images/Envelop.png" alt="Envelop"></li>
-                    </ul>                                       
+                    <h3 class="logo">Okpara Blessing </h3>
+                    <div id="show">
+                        <ul id="style"   >
+                            <div id="cross-icon">
+                                <img onclick="hide()" class="style-li" src="/images/close-Icon.png" alt="close-Icon">
+                              </div>
```
Comment (ShadyShawkat) [inline index.html:25]: - [x] Kindly eliminate the horizontal scroll in your website which results in this excess white space appearing on the right for your project to be more professional.
image

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — APPROVED** [review]
Hi @Luchy-B 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Good looking readme file 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**